app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -110,6 +116,7 @@
 }
 
 .mortgagecalc__spread {
+  margin-bottom: 0.5rem;
```

Comment:
As this is vertical spacing should this be defined using `$baseline-unit` or does that not make sense in this case?